docs/source/api/plotting.rst

```diff
@@ -17,4 +17,8 @@ Plotting
 
 .. autoclass:: divHretention.plot_particle_exposure_along_divertor
    :members:
-   :show-inheritance:
\ No newline at end of file
+   :show-inheritance:
+
+.. autoclass:: divHretention.plot_along_divertor
```

Comment:
I know I put this in the example. But you're not trying to add an entry for the `plot_along_divertor` function but your new `plot_inv_with_uncertainty`
```suggestion
.. autoclass:: divHretention.plot_inv_with_uncertainty
```